UI/window-basic-settings-stream.cpp

```diff
@@ -1360,7 +1360,9 @@ static QString get_adv_fallback(const QString &enc)
 		return "h264_texture_amf";
 	if (enc == "com.apple.videotoolbox.videoencoder.ave.hevc")
 		return "com.apple.videotoolbox.videoencoder.ave.avc";
-	return "obs_x264";
+	if (enc == "obs_x264")
+		return "obs_x264";
+	return "ffmpeg_openh264";
```

Comment:
I think you misunderstood the function purpose, it fallbacks HEVC encoders to H264 (some improvement needs to be done like you can see in the comment earlier in the code).

Last resort fallback must be obs_x264 if available (and ffmpeg_openh264 if no x264).